seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py

```diff
@@ -0,0 +1,177 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@INPROCEEDINGS{8584876,
+  author={Joseph, Ferdin Joe John and Anantaprayoon, Panatchakorn},
+  booktitle={2018 International Conference on Information Technology (InCIT)},
+  title={Offline Handwritten Thai Character Recognition Using Single Tier Classifier and Local Features},
+  year={2018},
+  volume={},
+  number={},
+  pages={1-4},
+  abstract={Handwritten character recognition is a conversion process of handwriting into machine-encoded text. Currently,
+  several techniques and methods are proposed to enhance accuracy of handwritten character recognition for many languages
+  spoken across the globe. In this project, a local feature-based approach is proposed to enhance the accuracy of handwritten
+  offline character recognition for Thai alphabets. In the experiment, through MATLAB, 100 images for each class of Thai
+  alphabets are collected and k-fold cross validation is applied to manage datasets to train and test. A gradient invariant
+  feature set consisting of LBP and shape features is extracted. The classification is operated by using query matching based
+  on Euclidean distance. The accuracy would be the percentage of correct classification for each class. For the result, the
+  highest accuracy is 68.96% which has 144-bit shape features and uniform pattern LBP for the features.},
+  keywords={Character recognition;Feature extraction;Shape;Genetic algorithms;Matlab;Gray-scale;Optical character recognition
+  software;Offline Character Recognition;Local Binary Pattern;Thai Handwriting},
+  doi={10.23919/INCIT.2018.8584876},
+  ISSN={},
+  month={Oct},
+  url={https://ieeexplore.ieee.org/document/8584876}}
+
+"""
+
+_DATASETNAME = "kvis_th_ocr"
+
+_DESCRIPTION = """\
+The KVIS Thai OCR Dataset contains scanned handwritten version of all 44 Thai characters obtained from 27 individuals. It
+consisted of 1079 images from 44 classes (letters). This dataset consists of all Thai consonants with different writing
+styles of various people from ages between 16 and 75. Vowels and intonation are not taken into consideration for the dataset
+collected.
+"""
+
+_HOMEPAGE = "https://data.mendeley.com/datasets/8nr3pbdk5c/1"
+
+_LANGUAGES = ["tha"]
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+
+_URLS = "https://data.mendeley.com/public-files/datasets/8nr3pbdk5c/files/e5730ffa-527c-4e06-ac76-aab758f63ae0/file_downloaded"
```

Comment:
I ran the standard test script 

```
python -m tests.test_seacrowd seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py
```

But the download link for the data doesn't seem to work. Can you resolve this? Thank you!

Reply:
Hi @faridlazuarda, could you tell me a bit more about the issue? On my end, the datasets downloads as expected.

Reply:
Hello, this is what I got:

```
ConnectionError: Couldn't reach https://data.mendeley.com/public-files/datasets/8nr3pbdk5c/files/e5730ffa-527c-4e06-ac76-aab758f63ae0/file_downloaded (ConnectionError(ProtocolError('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))))

----------------------------------------------------------------------
Ran 1 test in 1.907s

FAILED (errors=1)
```

Reply:
It works perfectly now. Thanks!